.vscode/launch.json

```diff
@@ -1,18 +1,44 @@
 {
     "version": "0.2.0",
     "configurations": [
-
         {
-            "name": ".NET Core Launch (console)",
-            "type": "coreclr",
+            "type": "PowerShell",
+            "request": "launch",
+            "name": "Build",
+            "script": "./build.ps1",
+            "args": [
+                ""
+            ],
+            "cwd": "${workspaceFolder}",
+            "createTemporaryIntegratedConsole": true
+        },
+        {
+            "type": "PowerShell",
+            "request": "launch",
+            "name": "Build & Run all tests",
+            "script": "./build.ps1; ./build.ps1 -Test",
+            "args": [
+                ""
```

Comment:
The PowerShell extension doesn't like it if `args` is just an empty array, hence the empty string.